### DISCUSSION TITLE
Jax could not find GPU. This can be either because your machine doesn't have a GPU

### DISCUSSION THREAD

**Discussion by WHEREISSHE** [jax-ml/jax#10983] [Q&A]
When I tried to run `colabfold` in my local machine, the error occured----
```
2022-06-04 17:38:16.431497: E external/org_tensorflow/tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
ERROR: Jax could not find GPU. This can be either because your machine doesn't have a GPU
or because jax can't find it. You might need to run

pip install --upgrade "jax[cuda]" -f https://storage.googleapis.com/jax-releases/jax_releases.html  # Note: wheels only available on linux.
```
How could I fix it? 
My desktop had a RTX 3060 GPU, and run `nvcc --version` got message:
```
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2021 NVIDIA Corporation
Built on Thu_Nov_18_09:45:30_PST_2021
Cuda compilation tools, release 11.5, V11.5.119
Build cuda_11.5.r11.5/compiler.30672275_0
```

**Comment by YouJiacheng** [comment]
Do you run `pip install --upgrade "jax[cuda]" -f https://storage.googleapis.com/jax-releases/jax_releases.html`?

**Comment by WHEREISSHE** [comment]
Yeah，I did run this before，but the error still occurred.




在 2022年6月5日，16:07，YouJiacheng ***@***.***> 写道：

﻿

Do you run pip install --upgrade "jax[cuda]" -f https://storage.googleapis.com/jax-releases/jax_releases.html?

**Reply by MarjanGolshani** [reply]
Hi,
I have the same problem, could you please help me.

**Reply by WHEREISSHE** [reply]
> Hi, I have the same problem, could you please help me.

Sorry， bro！I still got no idea how to fix it. Maybe you could try to install it again from the very beginning.
